mozilla/contain-facebook
# Add subdomain support for custom sites to the container.

Fixes #567 

This PR adds subdomain support for any custom site added to the container. 

## Testing Steps

_(Note – You can add sites both from the in-page badge on marked Login with Facebook forms or through the panel, adding any site manually. The testing page I define below can be added through either way)_

1. Navigate to https://accounts.spotify.com/en/login 
1. Add site to Facebook Container (by whichever method) 
1. Open https://www.spotify.com/us/, which should be INSIDE the container. 
1. REMOVE either site from the container, should both stay removed. 

I recommend trying it on a few other sites – includes sites with no subdomain (even WWW).

Test sites (adding either site should include the other inside the container)
- Google.com (vs drive.google.com)
- Github.com (vs gist.github.com)
- Firefox.com (vs accounts.firefox.com)
- Amazon.co.uk (vs smile.amazon.co.uk)
